### ISSUE TITLE
Controller annotation

### ISSUE THREAD

**Issue by alex88** [ClickerMonkey/SemanticUI-Angular#17]
I've seen many controllers uses the form

```
controller: function($scope)
```

which creates issues when code is minified and angular refuses to load the controller in strict mode.

Shouldn't this use annotations like

```
controller: ['$scope', function($scope){}]
```

**Comment by ClickerMonkey** [comment]
Yes it should, this is one of the things I was working on earlier today. I'll make sure to update this issue when this is committed and released.

**Comment by alex88** [comment]
Awesome, thanks!

**Comment by ClickerMonkey** [comment]
Fixed with release `0.2.0`

**Comment by alex88** [comment]
Awesome! 🚀
